tst/EnergyPlus/unit/CMakeLists.txt

```diff
@@ -70,6 +70,7 @@ set( test_src
   ElectricPowerServiceManager.unit.cc
   EMSManager.unit.cc
   EvaporativeCoolers.unit.cc
+  EvaporativeFluidCoolers.unit.cc
```

Comment:
:+1: New unit test files are getting rare these days, this is a good missing piece.